contracts/consensus-gateway/ConsensusCogateway.sol

```diff
@@ -94,6 +95,83 @@ contract ConsensusCogateway is MasterCopyNonUpgradable, MessageBus, ConsensusGat
         );
     }
 
+    /**
+     * @notice Confirm deposit in order to mint tokens.
+     *
+     * @param _amount MOST token deposit amount in atto.
+     * @param _beneficiary Address of beneficiary on auxiliary chain.
```

Comment:
"... on target metachain" (moving away from auxiliary chain language with Mosaic-1, as the the state trie of the metachain is a full extention of the Ethereum now)

Reply:
Done 👍